domain/model/service/service.go

```diff
@@ -117,6 +121,13 @@ type State interface {
 
 	// UpdateCredential updates a model's cloud credential.
 	UpdateCredential(context.Context, coremodel.UUID, credential.Key) error
+
+	// GetActivatedModelUUIDs returns the subset of model uuids from the supplied list that are activated.
+	// If no model uuids are activated then an empty slice is returned.
+	GetActivatedModelUUIDs(ctx context.Context, uuids []string) ([]coremodel.UUID, error)
+
+	// GetAllActivatedModelsUUIDQuery returns the query string to get the activation status of all models.
```

Comment:
I think what we want:

`GetAllActivatedModelsUUIDQuery returns the database query to get all the activated models in the controller.'

Reply:
Thanks for pointing it out tom!